integration/e2e/e2e_test.go

```diff
@@ -914,3 +924,42 @@ func hashFile(file string) string {
 func chaincodeContainerNameFilter(n *nwo.Network, chaincode nwo.Chaincode) string {
 	return fmt.Sprintf("^/%s-.*-%s-%s$", n.NetworkID, chaincode.Label, hashFile(chaincode.PackageFile))
 }
+
+func channelParticipationList(n *nwo.Network, o *nwo.Orderer, expectedChannels []string, systemChannel ...string) {
```

Comment:
As we are going to use that a lot, how about creating a public utility method that returns the list, and another to match against expected values?

Reply:
I'd prefer to leave it private for now but I can change it if you insist. We can make it public as soon as we use it in another test.

Reply:
I think that the bulk of tests will not be located in the e2e package but in the raft package, or possibly in a package of their own. This is because most of the interesting functionality of of join and delete is only applicable to raft. But I have no problem in exposing that when we actually have a test that needs that.